app/models/user.js

```diff
@@ -1,16 +1,32 @@
+const bcrypt = require('bcryptjs');
+
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define(
-    'user',
+    'User',
     {
       id: { type: DataTypes.INTEGER, primaryKey: true, notNull: true, autoIncrement: true },
-      name: { type: DataTypes.STRING },
-      surname: { type: DataTypes.STRING },
+      firstName: { type: DataTypes.STRING },
+      lastName: { type: DataTypes.STRING },
       email: { type: DataTypes.STRING },
       password: { type: DataTypes.STRING }
     },
     {
-      timestamps: true
+      timestamps: true,
+      underscored: true
     }
   );
+
+  const encryptPassword = async userData => {
+    const salt = await bcrypt.genSalt(15);
```

Comment:
You should avoid using the magic numbers, I suggest to you put rounds number in an env var